.github/shared/src/spec-model.js

```diff
@@ -65,7 +73,29 @@ export class SpecModel {
             continue;
           }
 
-          const refs = await inputFile.getRefs();
+          let refs;
+          try {
+            refs = await inputFile.getRefs();
+          } catch (error) {
+            if (isResolverError(error)) {
+              const resolverError = /** @type {SpecError} */ {
+                message: `Spec Model encountered error resolving refs for input-file: ${inputFile.path}
+\tReadme: ${readme.path}
+\tTag: ${tag.name}
+\tError file: ${error.source}
+
+Ensure the input-file is valid and all refs are resolvable.`,
```

Comment:
The message contains formatting for better readability. I haven't previously done this but it keeps us out of the business of coupling calling code to error objects that Spec Model produces.